cmd/admin_user_create_test.go

```diff
@@ -117,4 +117,31 @@ func TestAdminUserCreate(t *testing.T) {
 		assert.Equal(t, 0, unittest.GetCount(t, &auth_model.AccessToken{}))
 		assert.ErrorContains(t, err, "access token does not have any permission")
 	})
+
+	createUserN := func(name string, args []string) error {
+		cmd := []string{"./gitea", "admin", "user", "create", "--username", name, "--email", name + "@gitea.local"}
+		cmd = append(cmd, args...)
+		return app.Run(cmd)
+	}
```

Comment:
It doesn't to test "space" behavior. You just still use the old `createUser`

The "space" behavior is guaranteed by the shell CLI

Reply:
The `string.Fields` call in the old createUser doesn't allow for spaces in arguments so it can't be used here.

Reply:
It doesn't to test "space" behavior. You could just still use the old `createUser`

The "space" behavior is guaranteed by the shell CLI

Reply:
If you still prefer to test the space behavior, it's better to refactor the old "createUser" to support arguments:

```
createUser := func(name, args ...string) error {
		return app.Run(append([]string{"./gitea", "admin", "user", "create", "--username", name, "--email", name + "@gitea.local"}, args...)...)
}
```

Reply:
I made some changes in https://github.com/go-gitea/gitea/pull/34241/commits/5f2aad7a67659572d001b692a863b576d7316f6b:

1. Refactor `createUser` to accept argument slice, then we can use it for general purpose, without duplicating code
2. Simplify the test cases, I think these 2 cases should be able to have a good coverage (same coverage as before)

Does it look good to you?

Reply:
Yes, that's definitely better, thank you!

Reply:
Thank you for the PR too! Will merge after CI passes.